.github/workflows/publish.yml

```diff
@@ -24,5 +24,5 @@ jobs:
     - name: Publish to PyPI
       uses: pypa/gh-action-pypi-publish@master
       with:
-        user: AllenCellModeling
+        user: aics
```

Comment:
oof we were just discussing this in software private and saying since there is only a single package published on `aics` we should convert everything over to `aicspypi`

Reply:
Ha! Will do for future packages.